core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerListener.java

```diff
@@ -21,6 +21,11 @@
  */
 public interface CircuitBreakerListener {
 
+    /**
+     * Invoked when the circuit breaker is initialized.
```

Comment:
Let's add a little description that the state is `Closed` when this is invoked.

Reply:
Added `Initial state is always CLOSED.`